### DISCUSSION TITLE
Hoppscotch in AWS ECS?

### DISCUSSION THREAD

**Discussion by zwolfinger** [hoppscotch/hoppscotch#4130] [Q&A]
Is anyone running Hoppscotch in AWS Elastic Container Services?  

We are preparing to make the attempt and are open to any insight, caveats, better options, etc.

**Comment by Zambonilli** [comment]
We've deployed Hoppscotch Community Edition in ECS. It was relatively straight forward, we created a service for each component and used the dockerhub images. From there, we used RDS with Postgres for the DB and ALB, route53 and cert manager for the DNS, TLS and routing.

The only part that is clunky is the DB migrations. We ended up using a bastion-host but there's no reason there couldn't be a task created with an overridden entrypoint/command.
